internal/utils/extract.go

```diff
@@ -58,7 +58,7 @@ func ExtractTarGz(gzipStream io.Reader, destDir string, stripPathComponents int)
 		switch header.Typeflag {
 		case tar.TypeDir:
 			d := filepath.Join(destDir, strippedName)
-			if err = os.Mkdir(filepath.Clean(d), 0o750); err != nil {
+			if err = os.MkdirAll(filepath.Clean(d), 0o750); err != nil {
```

Comment:
Avoid failing when creating targz root folder (ie: when `strippedName` is either empty or `/`).